src/dispatch/feedback/models.py

```diff
@@ -19,8 +22,11 @@ class Feedback(Base, TimeStampMixin):
 
 # Pydantic models
 class FeedbackBase(DispatchBase):
+    created_at: Optional[datetime]
     rating: str
     feedback: Optional[str]
+    incident: Optional[IncidentReadNested]
```

Comment:
Do we expect to have feedback not tied to an incident?

Reply:
No, the incident provides the context to help us understand the feedback.

Reply:
So why is it optional?

Reply:
Must be a leftover from when I was testing adding feedback through the UI, which I will tackle in a separate PR.